integtest/spec/all_books_broken_link_detection_spec.rb

```diff
@@ -156,6 +156,12 @@
       include_examples 'there are broken links in kibana',
                        'en/fleet/master/not-a-fleet-page.html'
     end
+    describe 'when there is a broken APM link' do
+      include_context 'there is a kibana link', true,
+                      '${APM_DOCS}not-an-apm-page.html', true
+      include_examples 'there are broken links in kibana',
+                       'en/apm/get-started/master/not-an-apm-page.html'
```

Comment:
```suggestion
                       'en/apm/not-an-apm-page.html'
```

This should fix the failing test.